build.template

```diff
@@ -1,7 +1,112 @@
-//! Configuration settings for this Rhai build
+//! Fixed hashing seeds for stable hashing.
```

Comment:
I suggest the stable hashing code moved out of `build.rs` into the source tree (e.g. `src/config/hashing_stable.rs`).

The template can keep only the env-var `AHASH_SEED` value which is written to a generated file, say `src/config/hashing_env.rs`.